packages/psd/src/index.ts

```diff
@@ -10,6 +10,3 @@ export type {Guide} from "./interfaces";
 export {parse} from "./methods";
 
 export default Psd;
-
-// This value will be generated by vite-plugin-top-level-await
-export declare let init: Promise<void>;
```

Comment:
While editing the `index.ts` file, it would be good to organize unnecessary exported objects.

How about changing everything to `export type` except for the PSD object and the parse method?

Reply:
This will be good for another PR

Reply:
We'll also have to change this code example in the README:

```ts
import Psd, {Group, Layer, Node} from "@webtoon/psd";

// Recursively traverse layers and layer groups
function traverseNode(node: Node) {
  if (node instanceof Group) {
    for (const child of node.children) {
      traverseNode(child);
    }
  } else if (node instanceof Layer) {
    // Do something with layer
  } else {
    throw new Error("Invalid node type");
  }
}

for (const node of psdFile.children) {
  traverseNode(node);
}
```

(specifically, `instanceof Group` and `instanceof Layer` will no longer work)

Reply:
#29